# Finnish translation for apport
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2009-12-22 23:59+0100\n"
"PO-Revision-Date: 2009-12-19 17:48+0000\n"
"Last-Translator: Timo Jyrinki <Unknown>\n"
"Language-Team: Finnish <fi@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2009-12-22 20:36+0000\n"
"X-Generator: Launchpad (build Unknown)\n"

#. TRANS: %s is the name of the operating system
#: ../apport/ui.py:74
#, python-format
msgid "This is not a genuine %s package"
msgstr "Tämä ei ole alkuperäinen %s-paketti"

#: ../apport/ui.py:96
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"Järjestelmässä on asennettuna joidenkin pakettien vanhentuneita versiota. "
"Päivitä seuraavat paketit ja tarkista, uusiutuuko ongelma:\n"
"\n"
"%s"

#: ../apport/ui.py:102
msgid ""
"The program crashed on an assertion failure, but the message could not be "
"retrieved. Apport does not support reporting these crashes."
msgstr ""
"Ohjelma kaatui väitevirheen vuoksi, mutta viestiä ei voitu hakea. Apport ei "
"tue näiden kaatumisten raportointia."

#: ../apport/ui.py:189
msgid "unknown program"
msgstr "tuntematon sovellus"

#: ../apport/ui.py:190
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "Ikävä kyllä ohjelma \"%s\" sulkeutui yllättäen"

#: ../apport/ui.py:191 ../apport/ui.py:258 ../apport/ui.py:364
#, python-format
msgid "Problem in %s"
msgstr "Ongelma osan %s kanssa"

#: ../apport/ui.py:192
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"Muistia ei ole vapaana tarpeeksi ongelman automaattiseen analysointiin ja "
"virheraportin lähettämiseen."

#. package does not exist
#: ../apport/ui.py:219 ../apport/ui.py:245 ../apport/ui.py:251
#: ../apport/ui.py:283 ../apport/ui.py:291 ../apport/ui.py:356
#: ../apport/ui.py:534 ../apport/ui.py:737 ../apport/ui.py:901
#: ../apport/ui.py:905 ../apport/ui.py:924 ../apport/ui.py:930
#: ../apport/ui.py:1477 ../apport/ui.py:1494 ../apport/ui.py:1981
#: ../apport/ui.py:1997 ../apport/ui.py:2008 ../apport/ui.py:2019
msgid "Invalid problem report"
msgstr "Virheellinen ongelmaraportti"

#: ../apport/ui.py:220 ../apport/ui.py:252
msgid "The report belongs to a package that is not installed."
msgstr "Raportti liittyy pakettiin, jota ei ole asennettu."

#: ../apport/ui.py:247 ../apport/ui.py:907
msgid "This problem report is damaged and cannot be processed."
msgstr "Tämä ongelman raportti on vahingoittunut, eikä sitä voida käsitellä."

#: ../apport/ui.py:259 ../apport/ui.py:365
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"Ongelman raportoiminen ei onnistunut:\n"
"\n"
"%s"

#: ../apport/ui.py:284
msgid "You are not allowed to access this problem report."
msgstr "Sinulla ei ole lukuoikeuksia tähän ongelmaraporttiin."

#: ../apport/ui.py:287
msgid "Error"
msgstr "Virhe"

#: ../apport/ui.py:288
msgid "There is not enough disk space available to process this report."
msgstr "Levyllä ei ole tarpeeksi tilaa raportin käsittelyyn."

#: ../apport/ui.py:316
msgid "No package specified"
msgstr "Pakettia ei ole määritelty"

#: ../apport/ui.py:317
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr ""
"Paketti tai PID-tunniste on annettava. Katso lisätietoja käyttämällä "
"valitsinta --help."

#: ../apport/ui.py:327
msgid "Invalid PID"
msgstr "PID-numero ei kelpaa"

#: ../apport/ui.py:328
msgid "The specified process ID does not belong to a program."
msgstr "Määritetty prosessin ID-numero ei kuulu kyseiselle ohjelmalle."

#: ../apport/ui.py:336
msgid "Permission denied"
msgstr "Lupa evätty"

#: ../apport/ui.py:337
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"Kyseinen prosessi ei kuulu sinulle. Aja tämä ohjelma prosessin omistajana "
"tai pääkäyttäjänä."

#: ../apport/ui.py:357
#, python-format
msgid "Package %s does not exist"
msgstr "Pakettia %s ei ole olemassa"

#: ../apport/ui.py:389 ../apport/ui.py:435 ../apport/ui.py:451
msgid "Updating problem report"
msgstr ""

#: ../apport/ui.py:390
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""

#: ../apport/ui.py:399
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-bug"
"\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""

#: ../apport/ui.py:436 ../apport/ui.py:452
msgid "No additional information collected."
msgstr ""

#: ../apport/ui.py:491
msgid "What kind of problem do you want to report?"
msgstr "Millainen ongelma raportoidaan?"

#: ../apport/ui.py:508
msgid "Unknown symptom"
msgstr "Tuntematon ongelma"

#: ../apport/ui.py:509
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "Ongelmaa \"%s\" ei tunnistettu."

#: ../apport/ui.py:551
msgid "%prog [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""
"%prog [valinnat] [ongelma|pid|paketti|ohjelman polku|.apport/."
"kaatumisraportti]"

#: ../apport/ui.py:553
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""

#: ../apport/ui.py:556
msgid "Start in bug updating mode. Can take an optional --package."
msgstr ""

#: ../apport/ui.py:559
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""

#: ../apport/ui.py:562
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""

#: ../apport/ui.py:565
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information. (Implied if pid is given as only "
"argument.)"
msgstr ""

#: ../apport/ui.py:568
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""

#: ../apport/ui.py:571
msgid "Print the Apport version number."
msgstr "Tulosta Apport-versionumero."

#: ../apport/ui.py:738
msgid "Could not determine the package or source package name."
msgstr "Paketin tai lähdekoodipaketin nimeä ei voi määrittää."

#: ../apport/ui.py:753
msgid "Unable to start web browser"
msgstr "Selaimen käynnistäminen epäonnistui"

#: ../apport/ui.py:754
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "Nettiselainta ei saatu käynnistettyä avattaessa %s."

#: ../apport/ui.py:860
#, python-format
msgid "Please enter your account information for the %s bug tracking system"
msgstr ""

#: ../apport/ui.py:871
msgid "Network problem"
msgstr "Verkko-ongelma"

#: ../apport/ui.py:872
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr ""
"Kaatumistietokantaan ei voi yhdistää. Tarkista Internet-yhteyden toimivuus."

#: ../apport/ui.py:896
msgid "Memory exhaustion"
msgstr "Muisti ei riitä"

#: ../apport/ui.py:897
msgid "Your system does not have enough memory to process this crash report."
msgstr ""
"Järjestelmässäsi ei ole tarpeeksi muistia tämän kaatumisraportin käsittelyyn."

#: ../apport/ui.py:920 ../apport/ui.py:929
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr "Tämä ongelmaraportti kytkeytyy ohjelmaan, joka ei ole enää asennettu."

#: ../apport/ui.py:961
msgid "Problem already known"
msgstr "Ongelma on jo tiedossa"

#: ../apport/ui.py:962
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"Tämä ongelma raportoitiin jo virheraportissa, joka on näkyvissä selaimessa. "
"Olkaa hyvä ja tarkastakaa, voitteko lisätä mitään tietoa, joka voisi auttaa "
"kehittäjiä."

#: ../bin/apport-cli.py:71
msgid "Press any key to continue..."
msgstr "Paina mitä tahansa näppäintä jatkaaksesi..."

#: ../bin/apport-cli.py:78
msgid "What would you like to do? Your options are:"
msgstr "Kuinka haluat edetä? Vaihtoehdot ovat:"

#: ../bin/apport-cli.py:82
#, python-format
msgid "Please choose (%s):"
msgstr "Ole hyvä ja valitse (%s):"

#. translators: first %s: application name, second %s: date, third %s: time
#: ../bin/apport-cli.py:139
#, python-format
msgid "%s closed unexpectedly on %s at %s."
msgstr "%s sulkeutui yllättäen %s:lla %s:ssa."

#: ../bin/apport-cli.py:143
msgid ""
"If you were not doing anything confidential (entering passwords or other\n"
"private information), you can help to improve the application by reporting\n"
"the problem."
msgstr ""
"Jos et ollut tekemässä mitään luottamuksellista (kuten syöttänyt salasanoja "
"tai muuta\n"
"henkilökohtaista tietoa), voit auttaa ohjelman kehittämisessä raportoimalla\n"
"ongelman."

#: ../bin/apport-cli.py:146 ../bin/apport-cli.py:165 ../bin/apport-cli.py:183
msgid "&Report Problem..."
msgstr "&Raportoi ongelma..."

#: ../bin/apport-cli.py:147
msgid "Cancel and &ignore future crashes of this program version"
msgstr "Peru ja &jätä jatkossa huomiotta tämän ohjelmaversion kaatumiset"

#: ../bin/apport-cli.py:148 ../bin/apport-cli.py:166 ../bin/apport-cli.py:184
#: ../bin/apport-cli.py:233 ../bin/apport-cli.py:314 ../bin/apport-cli.py:346
#: ../bin/apport-cli.py:367
msgid "&Cancel"
msgstr "&Peruuta"

#: ../bin/apport-cli.py:163
#, python-format
msgid "The package \"%s\" failed to install or upgrade."
msgstr "Paketin \"%s\" asennus tai päivitys epäonnistui."

#: ../bin/apport-cli.py:164 ../gtk/apport-gtk.ui.h:22
msgid ""
"You can help the developers to fix the package by reporting the problem."
msgstr "Voi auttaa kehittäjiä korjaamaan ongelman raportoimalla ongelmasta."

#: ../bin/apport-cli.py:176
msgid "Your system encountered a serious kernel problem."
msgstr "Järjestelmässä tapahtui vakava ydinongelma."

#: ../bin/apport-cli.py:180
msgid "You can help the developers to fix the problem by reporting it."
msgstr "Voit auttaa kehittäjiä korjaaman ongelman ilmoittamalla siitä."

#: ../bin/apport-cli.py:195
msgid "Send this data to the developers?"
msgstr ""

#: ../bin/apport-cli.py:197
msgid "Send problem report to the developers?"
msgstr "Lähetetäänkö raportti ongelmasta kehittäjille?"

#: ../bin/apport-cli.py:198
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"Kun raportti ongelmasta on lähetetty, ole ystävällinen ja täytä selaimeesi\n"
"automaattisesti aukeava lomake."

#: ../bin/apport-cli.py:217
msgid "(binary data)"
msgstr "(binääritiedot)"

#: ../bin/apport-cli.py:221
#, python-format
msgid "&Send complete report (recommended; %s)"
msgstr "&Lähetä täydellinen raportti (suositeltavaa; %s)"

#: ../bin/apport-cli.py:223
#, python-format
msgid "Send &reduced report (slow Internet connection; %s)"
msgstr "Lähetä &suppea raportti (hitaille Internet-yhteyksille; %s)"

#: ../bin/apport-cli.py:226
#, python-format
msgid "&Send report (%s)"
msgstr "&Lähetä raportti (%s)"

#: ../bin/apport-cli.py:230
msgid "&View report"
msgstr "&Tarkastele raporttia"

#: ../bin/apport-cli.py:231
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr ""
"&Säilytä raporttitiedosto myöhemmin lähetettäväksi tai muualle kopioitavaksi"

#: ../bin/apport-cli.py:263
msgid "Problem report file:"
msgstr "Ongelmaraporttitiedosto:"

#: ../bin/apport-cli.py:271 ../bin/apport-cli.py:276
msgid "&Confirm"
msgstr "&Vahvista"

#: ../bin/apport-cli.py:275
#, python-format
msgid "Error: %s"
msgstr "Virhe: %s"

#: ../bin/apport-cli.py:281
msgid "Collecting problem information"
msgstr "Kerätään tietoja ongelmasta"

#: ../bin/apport-cli.py:282
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"Kerätyt tiedot voidaan lähettää kehittäjille ohjelman parantamiseksi.\n"
"Tähän voi mennä muutama minuutti."

#: ../bin/apport-cli.py:294 ../gtk/apport-gtk.ui.h:21
msgid "Uploading problem information"
msgstr "Lähetetään tiedot ongelmasta"

#: ../bin/apport-cli.py:295
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"Kerätyt tiedot lähetetään virheseurantajärjestelmään.\n"
"Tähän voi mennä muutama minuutti."

#: ../bin/apport-cli.py:345
msgid "&Done"
msgstr "&Valmis"

#: ../bin/apport-cli.py:351
msgid "none"
msgstr "ei mitään"

#: ../bin/apport-cli.py:352
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "Valittu: %s. Useita valintoja:"

#: ../bin/apport-cli.py:368
msgid "Choices:"
msgstr "Valinnat:"

#: ../bin/apport-cli.py:382
msgid "Path to file (Enter to cancel):"
msgstr "Polku tiedostoon (Enter peruu):"

#: ../bin/apport-cli.py:387
msgid "File does not exist."
msgstr "Tiedostoa ei ole olemassa."

#: ../bin/apport-cli.py:389
msgid "This is a directory."
msgstr "Tämä on hakemisto."

#: ../bin/apport-cli.py:395
msgid "To continue, you must visit the following URL:"
msgstr "Jatkaaksesi sinun tulee käydä seuraavassa URL-osoitteessa:"

#: ../bin/apport-cli.py:397
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"Voit nyt käynnistää selaimen tai kopioida tämän URL-osoitteen selaimeen "
"toisella tietokoneella."

#: ../bin/apport-cli.py:399
msgid "Launch a browser now"
msgstr "Käynnistä selain nyt"

#: ../bin/apport-cli.py:406
msgid "No pending crash reports. Try --help for more information."
msgstr ""
"Ei odottavia kaatumisraportteja. Kokeile --help saadaksesi lisäinformaatiota."

#: ../data/apportcheckresume.py:58
msgid ""
"This occured during a previous suspend and prevented it from resuming "
"properly."
msgstr ""
"Tämä tapahtui viimeisen keskeytyksen aikana ja esti sitä jatkamasta "
"asianmukaisesti."

#: ../data/apportcheckresume.py:60
msgid ""
"This occured during a previous hibernate and prevented it from resuming "
"properly."
msgstr ""
"Tämä tapahtui viimeisen horrostilan aikana ja esti sitä jatkamasta "
"asianmukaisesti."

#: ../data/apportcheckresume.py:65
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""

#: ../gtk/apport-gtk.ui.h:1
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>Kerätään tietoja ongelmasta</b></big>"

#: ../gtk/apport-gtk.ui.h:2
msgid ""
"<big><b>Send problem report to the developers?</b></big>\n"
"\n"
"After the problem report has been sent, please fill out the form in the "
"automatically opened web browser."
msgstr ""
"<big><b>Lähetetäänkö ongelmaraportti sovelluksen kehittäjille?</b></big>\n"
"\n"
"Ole hyvä ja täytä automaattisesti selaimeen auennut lomake, kun "
"ongelmaraportti on lähetetty."

#: ../gtk/apport-gtk.ui.h:5
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>Lähetetään tietoja ongelmasta</b></big>"

#: ../gtk/apport-gtk.ui.h:7
#, no-c-format
msgid ""
"<span weight=\"bold\" size=\"larger\">Sorry, the package \"%s\" failed to "
"install or upgrade.</span>"
msgstr ""
"<span weight=\"bold\" size=\"larger\">Paketin \"%s\" asennus tai päivitys "
"epäonnistui.</span>"

#: ../gtk/apport-gtk.ui.h:8
msgid "Application problem"
msgstr "Ongelma sovelluksessa"

#: ../gtk/apport-gtk.ui.h:10
#, no-c-format
msgid "Complete report (recommended; %s)"
msgstr "Täydellinen raportti (suositus; %s)"

#: ../gtk/apport-gtk.ui.h:11
msgid "Content of the report"
msgstr "Raportin sisältö"

#: ../gtk/apport-gtk.ui.h:12
msgid ""
"If you were not doing anything confidential (entering passwords or other "
"private information), you can help to improve the application by reporting "
"the problem."
msgstr ""
"Jos et tehnyt mitään luottamuksellista (syöttänyt salasanoja tai muita "
"henkilökohtaisia tietoja), voit auttaa kehittämään ohjelmaa raportoimalla "
"ongelmasta."

#: ../gtk/apport-gtk.ui.h:13
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr "Kerätään tietoja ongelman korjaamisen helpottamiseksi."

#: ../gtk/apport-gtk.ui.h:14
msgid "Kernel problem"
msgstr "Ongelma käyttöjärjestelmän ytimessä"

#: ../gtk/apport-gtk.ui.h:15
msgid "Package problem"
msgstr "Ongelma paketissa"

#: ../gtk/apport-gtk.ui.h:17
#, no-c-format
msgid "Reduced report (slow Internet connection; %s)"
msgstr "Pelkistetty raportti (hidas Internet-yhteys; %s)"

#: ../gtk/apport-gtk.ui.h:18
msgid "Restart _Program"
msgstr "Uudelleenkäynnistä _ohjelma"

#: ../gtk/apport-gtk.ui.h:19
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"Kerättyjä tietoja lähetetään vianseurantajärjestelmään. Tämä saattaa kestää "
"muutamia minuutteja."

#: ../gtk/apport-gtk.ui.h:20
msgid ""
"This will remove some large items from the report. These are very useful for "
"developers to debug the problem, but might be too big for you to upload if "
"you have a slow internet connection."
msgstr ""
"Tämä poistaa joitain isoimpia osuuksia raportista. Osuudet ovat hyödyllisiä "
"ohjelman kehittäjille ongelman selvittämisessä mutta et välttämättä halua "
"odottaa niiden lähettämistä, jos Internet-yhteytesi on hidas."

#: ../gtk/apport-gtk.ui.h:23
msgid "_Ignore future crashes of this program version"
msgstr "_Jätä tämän ohjelmaversion kaatumiset huomiotta jatkossa"

#: ../gtk/apport-gtk.ui.h:24
msgid "_Report Problem..."
msgstr "_Raportoi ongelma..."

#: ../gtk/apport-gtk.ui.h:25
msgid "_Send Report"
msgstr "_Lähetä raportti"

#: ../bin/apport-retrace.py:28
msgid "%prog [options] <apport problem report | crash ID>"
msgstr ""

#: ../bin/apport-retrace.py:30
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr ""

#: ../bin/apport-retrace.py:33
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""
"Aloita interaktiiviinen gdb-istunto raportin ydinvedoksen kanssa (-o jättää "
"huomioimatta; ei kirjoita raporttia)"

#: ../bin/apport-retrace.py:36
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr ""
"Kirjoita muokattu raportti annetulle tiedostolle alkuperäisen raportin "
"muuttamisen sijaan."

#: ../bin/apport-retrace.py:39
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr ""

#: ../bin/apport-retrace.py:42
msgid "Override report's CoreFile"
msgstr ""

#: ../bin/apport-retrace.py:45
msgid "Override report's ExecutablePath"
msgstr ""

#: ../bin/apport-retrace.py:48
msgid "Override report's ProcMaps"
msgstr ""

#: ../bin/apport-retrace.py:51
msgid "Rebuild report's Package information"
msgstr "Rakenna raportin Paketti-informaatio uudelleen"

#: ../bin/apport-retrace.py:54
msgid "Report download/install progress when installing additional packages"
msgstr "Raportoi latauksen/asennuksen edistyminen asennettaessa lisäpaketteja"

#: ../bin/apport-retrace.py:57
msgid ""
"Only unpack the additionally required packages, do not configure them; purge "
"packages again after retracing"
msgstr ""
"Ainoastaan pura tarvittavat lisäpaketit; älä konfiguroi niitä; puhdista "
"paketit uudestaan seurannan jälkeen"

#: ../bin/apport-retrace.py:60
msgid ""
"Do not use packaging system when using -u and do not purge packages "
"afterwards. This should only be used for temporarily unpacked chroot "
"tarballs where it would just be a waste of time."
msgstr ""
"Älä käytä pakettijärjestelmää käytettäessä -u -komentoa. Älä myöskään "
"puhdista paketteja jälkeenpäin. Tätä parametria tulisi käyttää ainostaan "
"väliaikaisiin pakkamattomiin chroot tarball:eihin, joissa se olisi vain "
"ajanhukkaa."

#: ../bin/apport-retrace.py:63
msgid "Install an extra package (can be specified multiple times)"
msgstr "Asenna ylimääräinen paketti (voidaan määritellä useita kertoja)"

#: ../bin/apport-retrace.py:66
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""

#: ../bin/apport-retrace.py:69
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""
"Näytä seuratut pinojäljet ja pyydä vahvistusta ennen niiden lähettämistä "
"kaatumistietokantaan."

#: ../bin/apport-retrace.py:72
msgid "Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr ""
"Polku jäljennös-sqlite-tietokantaan (oletus: ei jälkennöksen tarkistusta)"

#: ../bin/apport-retrace.py:78
msgid "incorrect number of arguments; use --help for a short help"
msgstr ""
"virheellinen lukumäärä argumentteja; käytä --help:iä saadaksesi lyhyesti apua"

#: ../bin/apport-retrace.py:82
msgid ""
"you either need to do a local operation (-s, -g, -o) or supply an "
"authentication file (--auth); see --help for a short help"
msgstr ""
"sinun tarvitsee tehdä joko paikallinen operaatio (-s, -g, -o) tai toimittaa "
"varmennustiedosto (--auth); katso --help saadaksesi lyhyesti apua"

#. translators: don't translate y/n, apport currently only checks for "y"
#: ../bin/apport-retrace.py:115
msgid "OK to send these as attachments? [y/n]"
msgstr "Sopiiko näiden lähettäminen liitteinä? [y/n] (y=kyllä, n=ei)"

#: ../gtk/apport-gtk-mime.desktop.in.h:1 ../kde/apport-kde.desktop.in.h:1
#: ../gtk/apport-gtk.desktop.in.h:1 ../kde/apport-kde-mime.desktop.in.h:1
msgid "Report a malfunction to the developers"
msgstr "Raportoi toimintahäiriöstä sovelluksen kehittäjille"

#: ../gtk/apport-gtk-mime.desktop.in.h:2 ../kde/apport-kde.desktop.in.h:2
#: ../gtk/apport-gtk.desktop.in.h:2 ../kde/apport-kde-mime.desktop.in.h:2
msgid "Report a problem..."
msgstr "Raportoi ongelma"

#: ../bin/apport-unpack.py:23
#, python-format
msgid "Usage: %s <report> <target directory>"
msgstr ""

#: ../bin/apport-unpack.py:33
msgid "Destination directory exists and is not empty."
msgstr "Kohdehakemisto on jo olemassa, eikä ole tyhjä."

#: ../data/kernel_oops.py:28
msgid "Your system might become unstable now and might need to be restarted."
msgstr ""
"Järjestelmästäsi saattaa tulla epävakaa ja se saattaa tarvita "
"uudelleenkäynnistystä."

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "Apport-kaatumistiedosto"

#~ msgid "The running kernel is not an Ubuntu kernel"
#~ msgstr "Käynnissä oleva ydin ei ole Ubuntu-ydin"
